OpenOversight/app/templates/partials/officer_incidents.html

```diff
@@ -0,0 +1,23 @@
+<h3>Incidents</h3>
+{% if officer.incidents %}
+    <ul class="list-group">
+    {% for incident in officer.incidents %}
+        <li class="list-group-item">
+        <h4>
+            <a href="{{ url_for('main.incident_api', obj_id=incident.id)}}">
+                Incident {{ incident.report_number }}
+            </a>
+            {% if current_user.is_administrator or (current_user.is_area_coordinator and current_user.ac_department_id == incident.department_id) %}
```

Comment:
Same question RE: `is_admin_or_coordinator`

Reply:
IIRC this one is slightly different than `is_admin_or_coordinator` but I'll double check to make sure.

Reply:
Yeah this condition checks against the incident department ID. I'm assuming that's probably the same as the officer department ID for the page but I'm not sure so I didn't want to change it.

Reply:
Ooh, I see the difference now :sweat_smile: thanks!